templates/web.socketed.template.yml

```diff
@@ -23,3 +23,9 @@ run:
      to: |
        listen unix:/shared/nginx.https.sock ssl;
        set_real_ip_from unix:;
+  - replace:
+     filename: "/etc/nginx/conf.d/outlets/before-server/10-redirect-http-to-https.conf"
```

Comment:
`20-https.conf` is only created in the `web.ssl.template.yml` so we suffer from a similar problem. If this template is used without the `web.ssl.template.yml`, this template will break.

Reply:
Is there a reason not to just always create the socket?